temba/flows/views.py

```diff
@@ -1647,6 +1648,29 @@ def get_context_data(self, *args, **kwargs):
 
             context["feature_filters"] = json.dumps(feature_filters)
 
+            # check if there is no other users that edititing current flow
+            # then make this user as main editor and set expiration time of editing to this user
+            password = getattr(settings, "REDIS_PW", None)
+            flow_editors_cache = redis.Redis(
```

Comment:
You can use the method `get_redis_connection` importing `from django_redis import get_redis_connection` to get the default Redis connection established on the settings file. E.g. https://github.com/greatnonprofits-nfp/rapidpro/blob/master/temba/campaigns/tasks.py#L25